### PR TITLE
Created composer.json

### PR DESCRIPTION
**Pull request by Dsazz**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    "require": {
+        "php": ">=5.5.0",
+        "symfony/framework-bundle": ">=2.1,<3.0",
+        "picqer/moneybird-php-client": "~0.9"
+    },
+
+    "autoload": {
+        "psr-0": {
+           "Kamilabs\\MoneyBirdApiBundle": ""
```
Comment (ama3ing) [inline composer.json]: I prefer Kami namespace

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        }
+    },
+
+    "target-dir": "Kamilabs/MoneyBirdApiBundle"
```
Comment (ama3ing) [inline composer.json]: Same here

File: Tests/KamiMoneyBirdBundle.php [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+ */
+
+
+namespace Kami\MoneyBirdApiBundle\Tests;
+
+use Kami\MoneyBirdApiBundle\KamiMoneyBirdBundle;
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+
+class PayumBundleTest extends \PHPUnit_Framework_TestCase
```
Comment (ama3ing) [inline Tests/KamiMoneyBirdBundle.php]: Payum? 🤣 

Call @makasim he will like this

File: .phpcomplete_extended/phpcomplete_index [hunk 4]
```diff
@@ -0,0 +1 @@
...:"(object $object ) : array","info":"(object $object ) : array"},{"word":"apache_child_terminate","kind":"f","menu":"( ) : bool","info":"( ) : bool"},{"word":"apache_get_modules","kind":"f","menu":"( ) : array","info":"( ) : array"},{"word":"apache_get_version","kind":"f","menu":"( ) : string","info":"( ) : string"},{"word":"apache_getenv","kind":"f","menu":"(string $variable ,[bool $walk_to_top] ) : string","info":"(string $variable ,[bool $walk_to_top] ) : string"},{"word":"apache_lookup_uri","kind":"f","menu":"(string $filename ) : object","info":"(string $filename ) : object"},{"word":"apache_note","kind":"f","menu":"(string $note_name ,[string $note_value] ) : string","info":"(string $note_name ,[string $note_value] ) : string"},{"word":"apache_request_headers","kind":"f","menu":"( ) : array","info":"( ) : array"},{"word":"apache_reset_timeout","kind":"f","menu":"( ) : bool","info":"( ) : bool"},{"word":"apache_response_headers","kind":"f","menu":"( ) : array","info":"( ) : array"},{"word":"apache_setenv","kind":"f","menu":"(string $variable ,string $value ,[bool $walk_to_top] ) : bool","info":"(string $variable ,string $value ,[bool $walk_to_top] ) : bool"},{"word":"apc_add",
```
Comment (ama3ing) [inline .phpcomplete_extended/phpcomplete_index]: This file shouldn't be here

File: .phpcomplete_extended/index_cache [hunk 5]
```diff
@@ -0,0 +1 @@
+{'namespace_cache': {}, 'classname_cache': {}, 'methods_cache': {}, 'fqcn_classkey_cache': {}}
```
Comment (ama3ing) [inline .phpcomplete_extended/index_cache]: This file shouldn't be here

**Review by ama3ing — CHANGES_REQUESTED** [review]
Don't forget to remove:

```.phpcomplete_extended/```

And ignore this please

File: Tests/KamiMoneyBirdBundle.php (hunk 3 above)
Comment (makasim) [reply Tests/KamiMoneyBirdBundle.php]: 👎

Comment (ama3ing) [reply Tests/KamiMoneyBirdBundle.php]: You are getting copied, that's mean how awesome you are 🥇

Comment (ama3ing) [reply Tests/KamiMoneyBirdBundle.php]: @Dsazz give credits to @makasim in this file.

**Review by ama3ing — APPROVED** [review]

**Pull request merged**